libpod/container_internal_linux.go

```diff
@@ -1197,13 +1197,13 @@ func (c *Container) makeBindMounts() error {
 		if c.config.NetNsCtr == "" {
 			if resolvePath, ok := c.state.BindMounts["/etc/resolv.conf"]; ok {
 				if err := os.Remove(resolvePath); err != nil && !os.IsNotExist(err) {
-					return errors.Wrapf(err, "error removing container %s resolv.conf", c.ID())
+					return errors.Wrapf(err, "error removing container from %s", c.ID())
```

Comment:
I think this shouldn't change, error doesn't make sense anymore.

Reply:
My goal here is not to have resolv.conf mentioned twice.

"error removing from container%s" would be better, then error would look like
error removing from container ABCDE: removing PATHTO/etc/resolv.conf permission denied
Or something like that.